events/TransferFamily/CreateServer.json

```diff
@@ -9,6 +9,27 @@
     "mitreAttackTechniques": [
         "T1537 - Transfer Data to Cloud Account"
     ],
+    "mitreAttackSubTechniques": [
+
+    ],
+    "unverifiedMitreAttackTechniques": [
+        {
+            "technique": "T1078 - Valid Accounts",
+            "reason": "The server creation process may involve generating or utilizing valid credentials, which can be leveraged by attackers to gain unauthorized access to the system."
+        },
+        {
+            "technique": "T1505 - Server Software Component",
+            "reason": "By creating a new server, attackers can install or manage software components that support malicious activities, such as maintaining persistence or running unauthorized code."
```

Comment:
I don't think this one would work here as the SFTP is just an S3. There is nothing else you can do.

Reply:
Removed.